bundler/lib/bundler/plugin.rb

```diff
@@ -4,11 +4,12 @@
 
 module Bundler
   module Plugin
-    autoload :DSL,        File.expand_path("plugin/dsl", __dir__)
-    autoload :Events,     File.expand_path("plugin/events", __dir__)
-    autoload :Index,      File.expand_path("plugin/index", __dir__)
-    autoload :Installer,  File.expand_path("plugin/installer", __dir__)
-    autoload :SourceList, File.expand_path("plugin/source_list", __dir__)
+    autoload :DSL,             File.expand_path("plugin/dsl", __dir__)
```

Comment:
Maintaining this alignment spacing forces all these lines to be rewritten when a new longer class name is added to the autoloads. 😞